src/nwb_conversion_tools/nwbconverter.py

```diff
@@ -146,10 +146,15 @@ def run_conversion(
 
         if metadata is None:
             metadata = self.get_metadata()
-        if conversion_options is None:
-            conversion_options = self.get_conversion_options()
         self.validate_metadata(metadata=metadata)
+
+        if conversion_options is None:
+            conversion_options = dict()
+        default_conversion_options = self.get_conversion_options()
+        conversion_options = dict_deep_update(default_conversion_options, conversion_options)
+
```

Comment:
is this ever deep? Would a simple `dict.update` work?

Reply:
It is just one level deep. One for each of the interfaces so I think is needed. Just to be specific, this how the output of `get_conversion_options`  looks for the current Giocomo conversion:
```python
{'SpikeGLXRecording': {'write_as': 'raw', 'es_key': 'ElectricalSeries_raw', 'stub_test': False}, 'SpikeGLXLFP': {'write_as': 'raw', 'es_key': 'ElectricalSeries_lfp', 'stub_test': False}, 'Behavior': {}}
```

A simple loop and update for every interface will do the job. Maybe that's preferable?

Reply:
oh right, one for each of the interfaces. I think the way you have it is fine

Reply:
Should have depth 2 at the NWBConverter level (depth 1 at interface level)

```python
my_conversion_options = dict(
    Interface1=dict(foo=bar),
    Interface2=dict(this=that),
)
```